### PR TITLE
Add moveTop command

### PR DESCRIPTION
**Pull request by maddie**

### CODE REVIEW

File: src/consts.js [hunk 1]
```diff
@@ -130,6 +131,7 @@ module.exports = {
         windowEventMinimize: "window.event.minimize",
         windowEventMove: "window.event.move",
         windowEventMoved: "window.event.moved",
+        windowEventMoveTop: "window.event.move.top",
```
Comment (asticode) [inline src/consts.js]: Could you change this to :

```
        windowEventMovedTop: "window.event.moved.top",
```

?

**Review by asticode — CHANGES_REQUESTED** [review]
Don't forget to update GO as well.

Write a comment in this PR so that I'm notified that you've made the change 👍

**Comment by maddie** [comment]
Both here and Go lib has been updated with proposed change

**Comment by asticode** [comment]
Thanks for the quick update 👍

**Pull request merged**